dash-info.yaml

```diff
@@ -1,10 +1,2612 @@
+                    ),
+                    htmlTbody(
+                      htmlTr(
+                        htmlTd("This is in the body of the table")
+                      )
+                    ),
+                    htmlTfoot(
+                      htmlTr(
+                        htmlTd("This is in the footer of the table")
+                      )
+                    )
+                  )
+                )
+                )
+              )
+            )
+
+            app$run_server()
+    - name: htmlTd
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+            library(dashCoreComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                "Within an htmlTable, individual cells can be made with htmlTd",
+                htmlBr(),
+                htmlTable(
+                  list(
+                    htmlTr(
+                      list(
+                        htmlTh("Header 1"),
+                        htmlTh("Header 2")
+                      )
+                    ),
+                    htmlTr(
+                      list(
+                        htmlTd("this is a cell"),
+                        htmlTd("this is another cell")
+                      )
+                    )
+                  )
+                )
+                )
+              )
+            )
+
+            app$run_server()
+    - name: htmlTemplate
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+            library(dashCoreComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                "You can create an HTML template to be populated later via js",
+                htmlBr(),
+                htmlTable(
+                  id = "myTable",
+                  htmlTr(
+                    list(
+                      htmlTh("Header 1"),
+                      htmlTh("Header 2")
+                    )
+                  )
+                ),
+                htmlTemplate(
+                  id = "myRowTemplate",
+                  htmlTr(
+                    list(
+                      htmlTd(className = "someRowValue"),
+                      htmlTd()
+                    )
+                  )
+                )
+                )
+              )
+            )
+
+            app$run_server()
+    - name: htmlTextarea
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+            library(dashCoreComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                htmlTextarea(
+                  rows = 4, cols = 50,
+                  children = "A text area allows users to input text"
+                )
+                )
+              )
+            )
+
+            app$run_server()
+    - name: htmlTfoot
+      do
```

Comment:
can we not wrap the strings in `htmlP`, just `"longstring", htmlWbr(), "secondpart"`? I don't think it does anything as is.

Reply:
This alternative:

```
app$layout(
  htmlDiv(list(
    "In a long string, it might be a good idea to add an htmlWbr to specify word breaks",
    "Thisverylongstringwithnowhitespaceswon'tlookverygood",
    htmlWbr(),
    "butatleastyoucanspecifya'natural'placeforthestringtobebrokenup"
  )
 )
)
```

yields

```
In a long string, it might be a good idea to add an htmlWbr to specify word breaksThisverylongstringwithnowhitespaceswon'tlookverygoodbutatleastyoucanspecifya'natural'placeforthestringtobebrokenup
```

while the original snippet appears to produce

```
In a long string, it might be a good idea to add an htmlWbr to specify word breaks
Thisverylongstringwithnowhitespaceswon'tlookverygood

butatleastyoucanspecifya'natural'placeforthestringtobebrokenup
```

Reply:
hmm not worth spending a lot of time on, but if you're already breaking the words into different `<p>` then what does it matter if you `<wbr>`? Seems like then it just acts as a `<br>`. But if I try it in Python using just strings or strings in `<span>`s:
```py
html.Div([
    "a" * 100,
    "b" * 100,
    html.Wbr(),
    "c" * 100,
    html.Wbr(),
    html.Span("d" * 100),
    html.Span("e" * 100)
])
```
I get what I expected:
<img width="1076" alt="Screen Shot 2020-01-09 at 9 21 09 PM" src="https://user-images.githubusercontent.com/2678795/72120454-faa2c800-3325-11ea-9a0b-fdf3c85fe836.png">

Reply:
Interesting, will merge this now but revisit this example later.